openapi/dev.to/Package.md

```diff
@@ -2,7 +2,7 @@ Connects to [DEV API v0.9.7](https://developers.forem.com/api/) from Ballerina.
 
 ## Package overview
 
-The `ballerinax/devto` is a [Ballerina](https://ballerina.io/) connector for DEV API. This API provides a access to forem articles, users and other resources. For a real-world example of Forem in action, check out [DEV](https://www.dev.to). All endpoints that don't require authentication are CORS enabled.
+The `ballerinax/dev.to` is a [Ballerina](https://ballerina.io/) connector for DEV API. This API provides a access to forem articles, users and other resources. For a real-world example of Forem in action, check out [DEV](https://www.dev.to). All endpoints that don't require authentication are CORS enabled.
```

Comment:
```suggestion
The `ballerinax/dev.to` is a [Ballerina](https://ballerina.io/) connector for DEV API. This API provides access to forum articles, users and other resources. For a real-world example of Forum in action, check out [DEV](https://www.dev.to). All endpoints that don't require authentication are CORS enabled.
```

Reply:
It seems `forem` is the word the vendor has been used on their official website. https://developers.forem.com/api